mappings/net/minecraft/block/PitcherCropBlock.mapping

```diff
@@ -9,8 +9,34 @@ CLASS net/minecraft/unmapped/C_fntbpikr net/minecraft/block/PitcherCropBlock
 	FIELD f_vspqavca UPPER_OUTLINE_SHAPES [Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_zixwbbhv AGE Lnet/minecraft/unmapped/C_vltzvhxi;
 	FIELD f_zotorzfo FULLY_GROWN_LOWER_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	METHOD m_duxcbgiw isUpperHalf (Lnet/minecraft/unmapped/C_txtbiemp;)Z
+		ARG 0 state
 	METHOD m_ecjdnzgk isMaxAge (Lnet/minecraft/unmapped/C_txtbiemp;)Z
+		ARG 1 state
 	METHOD m_itpfzjql grow (Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_txtbiemp;Lnet/minecraft/unmapped/C_hynzadkk;I)V
+		ARG 1 world
+		ARG 2 state
+		ARG 3 pos
 		ARG 4 amount
 	METHOD m_luiomuxe isLowerHalf (Lnet/minecraft/unmapped/C_txtbiemp;)Z
+		ARG 0 state
+	METHOD m_ocvvosbx hasEnoughLight (Lnet/minecraft/unmapped/C_eemzphbi;Lnet/minecraft/unmapped/C_hynzadkk;)Z
+		ARG 0 world
+		ARG 1 pos
+	METHOD m_ojkbyeir canGrow (Lnet/minecraft/unmapped/C_eemzphbi;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_txtbiemp;I)Z
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+		ARG 4 age
 	METHOD m_sploctuc canGrowInto (Lnet/minecraft/unmapped/C_eemzphbi;Lnet/minecraft/unmapped/C_hynzadkk;)Z
+		ARG 0 world
+		ARG 1 pos
+	METHOD m_vhxaczet getLowerHalf (Lnet/minecraft/unmapped/C_eemzphbi;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_txtbiemp;)Lnet/minecraft/unmapped/C_fntbpikr$C_jrcoboha;
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+	CLASS C_jrcoboha LowerHalfInfo
```

Comment:
```suggestion
	CLASS C_jrcoboha LowerHalf
```
I think the info suffix is fairly redundant here, this is passed around as *the* lower half

Reply:
Oh, I didn't want to cause ambiguities here since yeah, It isn't the block itself but rather, information about that block

Reply:
I'd much. rather see `LowerHalfState` then -- imo since it has a `BlockState` info doesn't work

Reply:
i think that was the mojmap name that i was worries about ;-;
oh well, i find the `State` suffix weird here since it isn't a BlockState, a FluidState or anything similar, but rather "hey here's a block state and coords, glhf"; wouldn't that be info about the lower half?
oh well, i don't know

Reply:
hm. I concede, the info name is fine